css/interface.css

```diff
@@ -39,6 +39,23 @@ body {
 }
 
 /* FORMS */
+.select-holder {
+  margin-left: 25px;
+  margin-top: 8px;
+}
+.select-proxy-display .hidden-select {
+  position: absolute;
+  top: -1px;
+  bottom: -1px;
+  left: 0;
+  right: 0;
+  cursor: pointer;
+  z-index: 0;
+  -webkit-appearance: none;
+  -moz-appearance: none;
+  -ms-appearance: none;
+  appearance: none;
+}
```

Comment:
Isn't this something we already have in `studio-ui-css`?

Reply:
@tonytlwu You are right, it shouldn't be here, not why I added back in.